server/user/user.controller.js

```diff
@@ -25,6 +27,33 @@ function get(req, res) {
   return res.json(req.user);
 }
 
+function login(req, res, next) {
+  User.findOne({ username: req.body.username }, (error, doc) => {
+    if (!doc) {
+      const error1 = new APIError(`Error: Account with username not found! ${req.body.username}`, httpStatus.UNAUTHORIZED);
```

Comment:
Leaking information about the existence of accounts via login is typically viewed as a minor security risk. But that's not a big deal for us for this project at this stage, so we'll let this through.

Reply:
Yes. Actually meant to change this anyway so thanks for the reminder!